src/Benchmarks/Configuration/Scenarios.cs

```diff
@@ -155,14 +155,14 @@ public Scenarios(IScenariosConfiguration scenariosConfiguration)
 
         public int Enable(string partialName)
         {
-            if(string.Equals(partialName, "[default]", StringComparison.OrdinalIgnoreCase))
+            if (string.Equals(partialName, "[default]", StringComparison.OrdinalIgnoreCase))
             {
                 EnableDefault();
                 return 2;
             }
 
             var props = typeof(Scenarios).GetTypeInfo().DeclaredProperties
-                .Where(p => string.Equals(partialName, "[all]", StringComparison.OrdinalIgnoreCase) || p.Name.StartsWith(partialName, StringComparison.OrdinalIgnoreCase))
+                .Where(p => string.Equals(partialName, "[all]", StringComparison.OrdinalIgnoreCase) || p.Name.IndexOf(partialName, StringComparison.OrdinalIgnoreCase) >= 0)
```

Comment:
We want to keep `StartsWith`: https://github.com/aspnet/benchmarks/commit/c22b982cafaf5049aead6c330bed09e779ce0a20

If anything I would update TechEmpower to also use `StartsWith`.